src/test/encoding/import-generated.sh

```diff
@@ -4,27 +4,33 @@ archive=$1
 
 [ -d "$archive" ] || echo "usage: $0 <archive>"
 
-ver=`./ceph-dencoder version`
+if [ -e ceph-dencoder ]; then
+    export PATH=${PATH}:$(dirname `realpath ceph-dencoder`)
+elif [ -e bin/ceph-dencoder ]; then
+    export PATH=${PATH}:$(dirname `realpath bin/ceph-dencoder`)
+fi
+
+ver=`ceph-dencoder version`
 echo "version $ver"
 
 [ -d "$archive/$ver" ] || mkdir "$archive/$ver"
 
 tmp1=`mktemp /tmp/typ-XXXXXXXXX`
 
 echo "numgen\ttype"
-for type in `./ceph-dencoder list_types`; do
+for type in `ceph-dencoder list_types`; do
 
     [ -d "$archive/$ver/objects/$type" ] || mkdir -p "$archive/$ver/objects/$type"
 
-    num=`./ceph-dencoder type $type count_tests`
+    num=`ceph-dencoder type $type count_tests`
     echo "$num\t$type"
     max=$(($num - 1))
     for n in `seq 0 $max`; do
-	./ceph-dencoder type $type select_test $n encode export $tmp1
-	md=`md5sum $tmp1 | awk '{print $1}'`
-	echo "\t$md"
-	[ -e "$archive/$ver/objects/$type/$md" ] || cp $tmp1 $archive/$ver/objects/$type/$md
+        ceph-dencoder type $type select_test $n encode export $tmp1
+        md=`md5sum $tmp1 | awk '{print $1}'`
```

Comment:
you are replacing tab with spaces. unless you are rewriting all of these lines, i'd suggest keep using whatever the "space" it was using.